vade/Makefile

```diff
@@ -23,6 +23,12 @@ VADEMAKEINTERNAL:=make -f $(VADEROOT)/vade/Makefile_internal SILENTMAKE=$(SILENT
 AR:=ar
 NM:=nm
 NASM:=nasm
+VALGRIND:=valgrind
+VGOPTS:=--exit-on-first-error=yes --error-exitcode=128
+VGOPTS$(L)+=-q
+VGOPTS+=--leak-check=full
+
+RUNTEST:=$(VALGRIND) $(VGOPTS)
```

Comment:
This is wrong: `valgrind` should be used only if the binary is available.
In this current form, that kills the CI because `valgrind` is not yet installed there (yet)

Reply:
Fixed. And added `valgrind` to the CI. Nice.